mappings/net/minecraft/inventory/InventoryChangeCallback.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_1265 net/minecraft/inventory/InventoryChangedListener
```

Comment:
1. The class name does not match the file name.
2. I think `InventoryChangeListener` would be better than `-Callback` since it fits well with other listener interfaces named `*Listener` in Yarn.

Reply:
@Juuxel Changed back to -Listener.

I'd originally gone with -Callback because I'd thought that was what we already used everywhere else.